src/components/DataTable.js

```diff
@@ -69,23 +123,36 @@ export default class DataTable extends React.Component {
         <table className={'table table-responsive ' + (wide ? styles.wide : styles.narrow)}>
           {(() => {
             if (wide) {
-              return (<thead>
-                <tr>
-                  {columns.map((col, i) => <th key={i}>{col}</th>)}
-                </tr>
-              </thead>);
+              return (
+                <thead>
+                  <tr>
+                    {columns.map((col, i) => {
+                      let arrow;
```

Comment:
That would create a nested ternary here, which the linter yelled at me for. I also feel like it makes it less readable, so I'm inclined to leave it as is unless you feel very strongly about it. Let me know!